docs/DeveloperGuide.adoc

```diff
@@ -1698,53 +1697,65 @@ Use case ends.
 .. 1a1. TA-Tracker creates a new session, and labels it as recurring.
 +
 Use case resumes at step 3.
+[none]
+. 1a. The user adds a session with a module code that does not exists.
+[none]
+.. 1a1. TA-Tracker shows an error message.
++
+Use case ends.
 
 [discrete]
-==== Use case: UC18 - Editing a session
+==== Use case: UC18 - Deleting a session
 
 *MSS*
 
 . User requests to go to the `Session Tab` (UC02) to view
 the list of existing sessions in the *Session View*.
 . TA-Tracker switches to the `Session Tab`.
-. User requests to edit a session.
-. TA-Tracker edits the session.
+. User requests to delete a session.
+. TA-Tracker deletes the session.
+. TA-Tracker shows the filtered sessions.
```

Comment:
This step is not needed right? Youre not filtering anything when you delete a session